toolchain/check/impl_lookup.cpp

```diff
@@ -66,6 +66,14 @@ static auto FindAssociatedImportIRs(Context& context,
           add_entity(context.interfaces().Get(SemIR::InterfaceId(arg)));
           break;
         }
+        case SemIR::IdKind::For<SemIR::FacetTypeId>: {
+          const auto& facet_type_info =
+              context.sem_ir().facet_types().Get(SemIR::FacetTypeId(arg));
```

Comment:
Should we expose `facet_types()` on `Check::Context` like we do for various other value stores?

Reply:
Probably, I can do that in a separate PR.